src/javascript/app/pages/user/account/settings/self_exclusion.js

```diff
@@ -268,10 +281,17 @@ const SelfExclusion = (() => {
                 });
             }
 
-            $('#gamstop_info_bottom').setVisibility(is_gamstop_client && date);
+            showWarning(timeout_val || exclude_until_val);
```

Comment:
I don't think this warning is needed to be displayed for `timeout_val`, is it?

Reply:
I'm not sure. I can’t see equal field in deriv.com but in old code it was showing a shorted warning text. Only first line of warning "Self-exclusion on the website only applies to your Binary.com account and does not include other companies or websites."

Reply:
you're right, my bad